packages/next/client/image.tsx

```diff
@@ -389,6 +401,8 @@ export default function Image({
     isLazy = false
   }
 
+  setRuntimeImageConfig(useContext(RuntimeImageConfigContext))
```

Comment:
We can put the context provider here instead:

https://github.com/vercel/next.js/blob/e3e62aa78673f8819583f56eb43c5cc9b4b9dd78/packages/next/client/index.tsx#L628

Reply:
Thanks a lot for your help!
But I may not understand what you mean?

<hr />

During SSR, There will only render `<AppContainer> from server/render.tsx`, 

but NOT render for `<AppContainer> from client/index.tsx`.

So, if I put `ImageConfigContext` to `<AppContainer> from client/index.tsx`, the error will still be thrown during SSR:

![7235abaf52187d87f1181bd77c55129](https://user-images.githubusercontent.com/14243906/152572723-93ad85a7-19d8-428f-bc35-81ead3445dc7.png)


Can I:
1. Skip `defaultLoader` config check during SSR? eg: `if (typeof window !== undefined)`.
2. Put `<RuntimeImageConfigContext.Provider>` to `<AppContainer> from server/render.tsx`.

What's your suggestion? Am I missing something?

Reply:
@JuniorTour Just to be clear, `ImageConfigContext` needs to included in **both** files:

- `next/server/render.tsx`
- `next/client/index.tsx`

Right around these lines:

https://github.com/vercel/next.js/blob/0b1d5e17bc304ba9bbc2cafc9b370bc66e359e7d/packages/next/server/render.tsx#L715

https://github.com/vercel/next.js/blob/e3e62aa78673f8819583f56eb43c5cc9b4b9dd78/packages/next/client/index.tsx#L628

Reply:
Got it.

I've add `<ImageConfigContext.Provider` to both `clent/index.tsx` and `server/render.tsx`.

Thank you again!